samples/logs.js

```diff
@@ -16,44 +16,50 @@
 
 async function writeLogEntry(logName) {
   // [START logging_write_log_entry]
-  // Imports the Google Cloud client library
   const {Logging} = require('@google-cloud/logging');
-
-  // Creates a client
   const logging = new Logging();
 
   /**
-   * TODO(developer): Uncomment the following line to run the code.
+   * TODO(developer): Uncomment the following line and replace with your values.
    */
-  // const logName = 'Name of the log to write to, e.g. my-log';
-
+  // const logName = 'my-log';
```

Comment:
If you don't want to have this as a comment, this should also work:

```
async function writeLogEntry(inputName) {
...
var logName = 'my-log';
[END logging_write_log_entry]
logName = inputName;
[START logging_write_log_entry]
...
```

The two halves of the region tag will be concatenated together, and the snippet will be both readable and testable